Moggles/ClientApp/src/featureToggle/EditFeatureToggle.vue

```diff
@@ -179,7 +182,10 @@
                     featureToggleName: this.rowToEdit.toggleName,
                     isPermanent: this.rowToEdit.isPermanent,
                     statuses: statuses,
-                    reasonToChange: !this.stringIsNullOrEmpty(this.reasonToChange) ? this.reasonToChange : null
+                    reasonToChange: {
+                        description: !this.stringIsNullOrEmpty(this.reasonToChange) ? this.reasonToChange : null,
+                        environments: this.environmentsToRefresh
```

Comment:
What happens when the toggle value was changed for the environment but the environment does not require a reason?

Reply:
you can save the feature toggle without adding a reason. If you want to add a reason, even if the environment does not require to add one, in the reason whill be added the name of the environment